paraatm/svm/_svm_text_classifier.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 28 08:49:23 2020
+
+@author: karvepm
+"""
+"""
+Support vector machine, multi-class, classifier for text data
+---
+Required pakages:
+Pipeline
+sklearn
+"""
+from sklearn.pipeline import Pipeline
+from sklearn.linear_model import SGDClassifier
+from sklearn.model_selection import GridSearchCV
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import TfidfTransformer
+
+
+class SVM_text_class(object):
+    """
+    SVM_text_class creates SVM classifier and predicts class for the input text data 
+    
+    ----------
+    x_train: training input
+    y_train: training target
+    x_test: testing input
+    
+    Methods:
+    -------
+    get_class
+        
+    """
+    def __init__(self,x_train,y_train):

```

Comment:
Please add a docstring to document the function parameters following our style guide, for example:
```suggestion
    def __init__(self, x_train, y_train):
        """
        Parameters
        ----------
        x_train : array
            2D array of training input values
        y_train : array
            1D array of training output values
        """
```